### ISSUE TITLE
Missing browserCriClient in connectToNewSpec

### ISSUE THREAD

**Issue by crfrolik** [cypress-io/cypress#29592]
### Current behavior

```
Timed out waiting for the browser to connect. Retrying...
Missing browserCriClient in connectToNewSpec
Error: Missing browserCriClient in connectToNewSpec
    at Object.connectToNewSpec (<embedded>:4438:48137)
    at Object.connectToNewSpec (<embedded>:4438:27043)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async v.relaunchBrowser (<embedded>:4504:37724)
```

### Test code to reproduce

It is not reproducible. I saw it happen once.

### Cypress Version

13.6.6

### Node version

v20

### Operating System

Ubuntu 22.04

**Comment by jennifer-shehane** [comment]
Duplicate of https://github.com/cypress-io/cypress/issues/27657

**Comment by gersoncdias** [comment]
I am facing the same problem, do we have any solution

**Comment by jennifer-shehane** [comment]
@gersoncdias This issue was resolved in 13.3.0. Please update that version and report any new issues encountered by opening a new issue.
